### PR TITLE
Jared

### PR DESCRIPTION
**Pull request by SpinksJared**

### CODE REVIEW

**Comment by DelgadoJosh** [comment]
Looks good to me

File: Campaign_Manager/templates/Campaign_Manager/campaign_confirm_deletion.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "Character_Builder/Character_base.html" %}
+
+{% block content %}
+    <div class="content-section">
+
+        <legend class="border-bottom mb-4">Delete Character</legend>
```
Comment (brhiner) [inline Campaign_Manager/templates/Campaign_Manager/campaign_confirm_deletion.html]: This should say "Delete Campaign"

**Comment by NickJacksonDev** [comment]
lgtm

File: Campaign_Manager/urls.py [hunk 2]
```diff
@@ -20,8 +20,10 @@
     path('campaigns/',
     	CampaignListView.as_view(), name='campaign-list'),
 
+    #path('campaigns/<int:pk>/',
```
Comment (brhiner) [inline Campaign_Manager/urls.py]: Do we need these comments, or can they be deleted?

**Review by brhiner — CHANGES_REQUESTED** [review]
Mostly looks good. Just delete a few comments and change the title for that one page and we should be good

**Review by brhiner — APPROVED** [review]

**Pull request merged**